test/package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "end2end-tests",
+  "private": true,
   "version": "0.0.0",
   "description": "This directory contains integration tests for the cdktf project.",
-  "scripts": {},
+  "scripts": {
+    "edge:install": "rm -rf ./edge-provider-bindings && cp -R ../packages/@cdktf/provider-generator/edge-provider-bindings/ ./edge-provider-bindings",
```

Comment:
could this be done via the installed packages in `./run-against-dist` ?

Reply:
Potentially 🤷  I don't feel super bold on bat so I thought I just take this approach

Reply:
Ok, so it's not used as part of the CI workflow, but a helper when doing this manually. That's probably ok then